.eslintignore

```diff
@@ -1 +1,2 @@
 mockServices
+src/scenes/home/faq/**/*
```

Comment:
I'm going to amend my commit, and let you guys figure out what you'd like to do.